tests/EntityManagerFactoryTest.php

```diff
@@ -77,7 +82,7 @@ class EntityManagerFactoryTest extends PHPUnit_Framework_TestCase
     /**
      * @var array
      */
-    protected $caches = [ 'query', 'result', 'metadata' ];
+    protected static $caches = [ 'query', 'result', 'metadata' ];
```

Comment:
Is there a reason this is static now?  Static properties in tests can lead to weird or incorrect test results.